client/src/App.css

```diff
@@ -1,40 +1,11 @@
 @import '~antd/dist/antd.css';
 
-/* Landing */
-.landing {
-  position: relative;
-  background: url('./img/landing_background.jpg') no-repeat center center/cover;
-  height: 100vh;
+.header {
```

Comment:
🔮 What thought do we need to put into name-spacing CSS styles?

Reply:
See [this issue](https://github.com/joshprzybyszewski/cribbage/issues/59#issue-630906998), I like the utility approach that tailwind provides (which removes the need for name spacing) and have been playing around with it.